app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,24 @@
+class SessionsController < ApplicationController
+  include SessionsHelper
+
+  skip_before_action :require_login
+
+  def new
+  end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user && user.password == params[:session][:password]
+      successful_session(user)
+    else
+      flash[:success] = 'Invalid email/password combination'
```

Comment:
You don't want to set a success flash key as an error... Is very backwards. You can set the flash key to something like `:error` instead.

Reply:
Fixed, thanks!